File: king-admin-java/src/main/java/com/oukingtim/domain/TbDictClass.java
package com.oukingtim.domain;

import com.baomidou.mybatisplus.annotations.TableName;
import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 * Created by oukingtim
 */
@TableName("tb_dict_class")
@Data
@EqualsAndHashCode(callSuper = false)
public class TbDictClass extends BaseModel<TbDictClass> {

    private String code;
    private String remark;

}

File: king-admin-kotlin/src/main/kotlin/com/oukingtim/domain/TbDictClass.kt
package com.oukingtim.domain

import com.baomidou.mybatisplus.annotations.TableName

/**
 * Created by oukingtim
 */
@TableName("tb_dict_class")
class TbDictClass : BaseModel<TbDictClass>() {
    val code: String? = null
    val remark: String? = null
}